src/cljs_ipfs_api/utils.cljs

```diff
@@ -73,24 +74,37 @@
                        f-n)))]
     callback))
 
-(defn is-buffer? [x]
-  (= Buffer (type x)))
+(defn is-buffer-or-blob? [x]
```

Comment:
Just a small tip (or personal preference) for the future - the boolean returning methods with question mark ending, I think `is-` is redundant. For me `buffer-or-blob?` would convey the meaning equally well but more briefly